scripts/release/README.md

```diff
@@ -1,24 +1,78 @@
-## Jenkins Release Build
+## Shared Directory Structure Pattern
+
+The `release/` directory will have a structure that looks like the following:
+
+        release/
+            build/
+            test/
+            prod/
+            ...
+            <scripts common to all builds {.sh,.md,Dockerfile,etc}>
+
+Each subdirectory of `release/` represents a build pipeline and is self-contained.  In other words, it will have its own `Jenkinsfile` and whatever else it needs to perform its job.
+
+The pattern is the following (all subdirecties of `release/` will follow this pattern):
+
+        test/                       // Maps to "test" build pipeline.
+            Jenkinsfile
+            stage/
+                setup/              // Maps to "setup" stage in Jenkinsfile.
+                    run.sh
+                    task.sh
+                test/               // Maps to "test" stage in Jenkinsfile.
+                    run.sh
+                    task.sh
+            deb/
+                *.sh
+                testDebian.exp
+            rpm/
+                *.sh
+
+The `Jenkinsfile` is self-explanatory, so I'll gloss over it.  `stage/` will contain directories that directly map to a defined stage in the `Jenkinsfile`.  Each directory will contain two files:
+
+1. run.sh
+1. task.sh
+
+`run.sh` will be the script called in the `Jenkinsfile` and contains any `aws cli` commands.  `task.sh` is then initiated by `run.sh` and contains the logic for the particular build stage.
+
+The `rpm` and `deb` directories will contain scripts that are relevant to their respective packaging formats.
+
+You may rely on this pattern, and this makes any troubleshooting relatively straightforward as you know where to find the code that executes a particular task due to setting reasonable expectations.
+
+## Build Outcomes
+
+This section briefly describes the expected outcomes of the current build pipelines.
+
+1. build
+
+    The result of running this job will be to put the build artifacts and their detached signatures in the staging `algorand-builds` bucket.
+
+    In addition, the build logs will be placed into the AWS `algorand-devops-misc` S3 bucket under `buildlog`.
```

Comment:
I think that I'd prefer to have all the pipeline output goes into the Algorand-builds directory.
however, from that point and on, I'm not over picky. If you would make it under ( let's say ) `algorand-builds/build-logs/channel/beta/` that would be fine.

Reply:
Sure, I can change that.  It's the old location.